ddtrace/internal/_encoding.pyx

```diff
@@ -534,6 +535,8 @@ cdef class MsgpackEncoderV03(MsgpackEncoderBase):
             L = len(d)
             if dd_origin is not NULL:
                 L += 1
+            if i == 0:
```

Comment:
comment on why would be good, less obvious here why we are increasing the length by 1 when `i` is `0`